src/custom-components/CustomDropdown.tsx

```diff
@@ -0,0 +1,141 @@
+import { SingleDropdownOption, DropdownProps, showContextMenu, Menu, MenuItem, showModal } from '../deck-components';
+import { ReactElement, VFC, useState, useEffect } from 'react';
+import { BsThreeDots } from 'react-icons/bs';
```

Comment:
We've used FA throughout decky, best to be consistent IMO

Reply:
all icons have been replaced with fa icons